### PR TITLE
Rename loss_fun to loss_class

### PR DESCRIPTION
**Pull request by markus-hinsche**
e.g. https://pytorch.org/docs/master/generated/torch.nn.MSELoss.html#torch.nn.MSELoss

### CODE REVIEW

**Comment by kbressem** [comment]
It is possible to pass a function as loss. Isn't loss function the more used term? In conversations about losses, I've never heard people use `loss_class`. Maybe call it just `loss`. This would be more consistent with `optimizer` and `model` which are also not called `model_class` or `optimizer_class`?

**Review by kbressem — APPROVED** [review]
